tracing/otlp.go

```diff
@@ -0,0 +1,35 @@
+package tracing
+
+import (
+	"go.opentelemetry.io/otel/exporters/otlp"
+	export "go.opentelemetry.io/otel/sdk/export/trace"
+	"google.golang.org/grpc/credentials"
+)
+
+// OTLP service to export traces to
+type OTLP struct {
+	Address string            `long:"otlp-address" description:"otlp address to send traces to"`
+	Headers map[string]string `long:"headers" description:"headers to attach to each tracing message"`
```

Comment:
i think this notion of headers is otlp-specific
```suggestion
	Headers map[string]string `long:"otlp-headers" description:"headers to attach to each tracing message"`
```